_posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md

```diff
@@ -0,0 +1,89 @@
+
+And speaking of words – they say "A picture is worth a thousand words". We think the same. This is why there is much more behind our tutorial than a simple prose with code samples. The tutorial is accompanied by two API Blueprint examples, its backend implementation tested in a continuous integration and finally, it is fully integrated within Apiary tools. 
+
+## Same problem, two solutions
+ Our new tutorial walks you though the basics of API Blueprint along with how you can build your very first blueprint. It uses an example of a polls service, which allows consumers to view polls and vote in them.
+ 
+ The tutorial is using a simplified Polls API version – without hypermedia controls and some advanced API Blueprint features.
+ 
+ The full version of this simplified Polls API can be found at <http://docs.pollsapi.apiary.io> or in the API Blueprint GitHub [repository](https://github.com/apiaryio/api-blueprint/blob/master/examples/Polls%20API.md).
+ 
+As we believe following REST constraints may bring competitive advantage in fields where API provider does not control clients we have also prepared a truly REST – hypermedia – version of the Polls API. 
+
+You can find the Hypermedia Polls API at <http://docs.pollshypermedia.apiary.io/> or in the API Blueprint GitHub [repository](https://github.com/apiaryio/api-blueprint/blob/master/examples/Polls%20Hypermedia%20API.md)
+
+This API offers both HAL and Siren hypermedia types through content-negotiation. If you are interested in details about hypermedia API design – please let us know on Twitter or in the discussion under this article – we will be happy to discuss it in-depth.
+
+## Backend implementation
+There is an [example implementation](http://github.com/apiaryio/polls-api) of the Polls API so you can perform requests to a real API.
+
+```bash
+$ curl https://polls.apiblueprint.org/questions/1
+```
+```json
+{
+    "question": "Favourite programming language?",
+    "published_at": "2014-11-11T08:40:51.620Z",
+    "url": "/questions/2",
+    "choices": [
+        {
+            "choice": "Swift",
+            "url": "/questions/1/choices/1",
+            "votes": 0
+        }, {
+            "choice": "Python",
+            "url": "/questions/1/choices/2",
+            "votes": 0
+        }
+    ]
+}
+```
+
+We've deployed an instance of this API and it's accessible for anyone to experiment with: `https://polls.apiblueprint.org/`. However you can also deploy your own instance of the Polls API to Heroku with a single click:
+
+[![Deploy to Heroku](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/apiaryio/polls-api)
+
+## CI tests
+At Apiary we care about the whole API lifecycle. This is why we have created [Dredd](https://blog.apiary.io/2013/10/10/No-more-outdated-API-documentation) – a testing tool for your blueprints and APIs. 
```

Comment:
So far we have always used it as one word (se API Blueprint repo and website). 

Also seem life cycle is going down – https://books.google.com/ngrams/graph?content=life+cycle%2C+lifecycle%2C+life-cycle&year_start=1800&year_end=2008&corpus=0&smoothing=3&direct_url=t1%3B%2Clife%20cycle%3B%2Cc0%3B.t1%3B%2Clifecycle%3B%2Cc0